login.php

```diff
@@ -0,0 +1,29 @@
+<?php  
+   require_once ("connection.php");        
```

Comment:
```suggestion
   require_once( "connection.php" );        
```

Reply:
parameters should have proper consistent formatting, please make changes in other places too.